modules/statics/src/base.ts

```diff
@@ -118,6 +118,10 @@ export enum CoinFeature {
    * This coin requires that accounts keep a minimum balance as reserve
    */
   REQUIRES_RESERVE = 'requires-reserve',
+  /**
+   * This coin supports custodial wallet types
+   */
+  CUSTODY = 'custody',
```

Comment:
this doesn't really fit here as I discussed earlier, but it doesn't fit in the same way that the `PAYGO` entry doesn't fit, so I guess I'll let it slide if it makes your life easier.

We should be wary of adding platform feature flags here - that's not the intended purpose.

Reply:
@FloBitGo do you still feel we should put this in statics instead of hardcoded in the UI(s) ?

Reply:
I thought the idea behind statics was that it would be our central coin configuration. I wasn't aware that it should be agnostic to platform. If that is the case, then we should probably discuss different solutions.

Reply:
The idea of `statics` is to avoid unnecessary duplication between BitGoJS and `platform`. So if both `platform` and `BitGoJS` need to know about this flag it belongs in here.

Reply:
yes, both platform & all the UIs will need to know about this flag so this makes the most sense then to define it here.